swagger/body-binary.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
is the reason this file appear completely different a formatting change?

Reply:
I wonder if it's because of the renaming?

Reply:
oh didn't see that